scripts/apps/search/components/actions-menu/MenuItems.tsx

```diff
@@ -95,7 +93,7 @@ export default class MenuItems extends React.Component<any, any> {
                     menu.push(
                         <li key={`group-label-${group._id}`}>
                             <SubmenuDropdown
-                                label={gettextCatalog.getString(group.label)}
+                                label={gettext(group.label)}
```

Comment:
Do not use `gettext` for dynamic values. Instead an alias of `gettext` should be used so the code inside `gettext` isn't extracted and sent to Transifex.

It doesn't look like it makes sense to translate this at all.

Reply:
We need translation for group.label.

Reply:
It should be translated above in the call tree, in this case in `scripts/apps/authoring/authoring/constants.js` where groups are defined.

The same applies to all comments relating to attempts at translating variables. Only static strings should be passed to `gettext`.